libft/sources/memory/ft_assign.c

```diff
@@ -1,16 +1,23 @@
 /* ************************************************************************** */
 /*                                                                            */
 /*                                                        :::      ::::::::   */
-/*   assign.c                                           :+:      :+:    :+:   */
+/*   ft_assign.c                                        :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
 /*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/08/13 22:58:25 by inikulin          #+#    #+#             */
-/*   Updated: 2024/08/17 10:38:19 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/09 23:29:54 by inikulin         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
-int	ft_assign(int *where, int what, int retval)
+int	ft_assign_i(int *where, int what, int retval)
+{
+	if (where)
+		*where = what;
+	return (retval);
```

Comment:
Interesting, what is idea to return retval passed to function and wasnt in use?

Reply:
instead of
```if (error)
{
    *errno = 5;
    return (0);
}```
we can do
```
if (error)
    ft_assign_i(errno, 5, 0);
```
this saving several lines, especially those damned curly braces